### DISCUSSION TITLE
openweathermap api key and wallpaper

### DISCUSSION THREAD

**Discussion by Weemizo** [rocksdanister/lively#1295] [General]
hello,
could somebody explain how the weather wallpaper works?
can i use other weather sites besides openweathermap ?
it looks like api key doesnt actually work..
![image](https://user-images.githubusercontent.com/60063669/178120814-9a374f32-d6aa-450f-8bdd-f4fcc5b9d3cf.png)

[i created an account, got an api key - [ctrl+c ctrl+s] into the file + added Wroclaw into string of cityName [though i could leave it blank, cause its not important]
![image](https://user-images.githubusercontent.com/60063669/178120692-161d3ac8-107f-4955-991a-8d8c452d0fff.png)

**Comment by rocksdanister** [comment]
Do not edit the code, API key is to be input in Customise menu which uses [LivelyProperties](https://github.com/rocksdanister/lively/wiki/Web-Guide-IV-:-Interaction#lively-properties):
<img width="330" alt="image" src="https://user-images.githubusercontent.com/17554161/178121100-7e04162e-996f-4778-addd-9b35c1490b47.png">

<img width="666" alt="image" src="https://user-images.githubusercontent.com/17554161/178121098-dd1379d9-036e-40d0-a427-a323fe0b0bd4.png">

Also keep in mind key takes time to activate on openweathermap side (day or two) so you will get error until its ready.

**Comment by Weemizo** [comment]
Seems to be working now, thanks!
